tfx_bsl/beam/run_inference.py

```diff
@@ -413,13 +415,28 @@ def _prepare_instances(
         if attr_name is None:
           continue
         attr = getattr(feature, attr_name)
-        values = cls._parse_feature_content(attr.value, attr_name,
-                                            cls._sending_as_binary(input_name))
+        values = self._parse_feature_content(attr.value, attr_name,
+                                             self._sending_as_binary(input_name))
         # Flatten a sequence if its length is 1
         values = (values[0] if len(values) == 1 else values)
         instance[input_name] = values
       yield instance
 
+  def _prepare_instances_serialized(
+      self, elements: List[tf.train.Example]
+  ) -> Generator[Mapping[Text, Any], None, None]:
+    """Prepare instances by base64 encoding serialized examples."""
+    for example in elements:
+      yield {'b64': base64.b64encode(example.SerializeToString()).decode()}
+
+  def _prepare_instances(
+      self, elements: List[tf.train.Example]
+  ) -> Generator[Mapping[Text, Any], None, None]:
+    if self._ai_platform_prediction_model_spec.use_serialization_config:
```

Comment:
if self. _ai_platform_prediction_model_spec.HasField("use_serialization_config") and self._ai_platform_prediction_model_spec.use_serialization_config

Reply:
This is not necessary because as mentioned earlier, default value of bool in proto is false. No need to check HasField. self._ai_platform_prediction_model_spec.use_serialization_config is sufficient